bucket/dottorrent-gui.json

```
@@ -0,0 +1,43 @@
+{
+    "version": "1.3.11",
+    "description": "An advanced GUI torrent file creator with batch functionality.",
+    "homepage": "https://github.com/kz26/dottorrent-gui",
+    "license": {
+        "identifier": "GPL-3.0-only",
+        "url": "https://github.com/kz26/dottorrent-gui/blob/master/LICENSE"
+    },
+    "suggest": {
+        "Microsoft Visual C++ Redistibutable 2015": "extras/vcredist2015"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kz26/dottorrent-gui/releases/download/v1.3.11/dottorrent-gui-1.3.11-win64.rar",
+            "hash": "4c40e001f4159c44007d6beaf3cfcd3b03fa9c6603fe8093ec754914beb00dda",
+            "extract_dir": "dottorrent-gui"
+        }
+    },
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\dottorrent-gui.ini\")) { New-Item \"$dir\\dottorrent-gui.ini\" | Out-Null }",
+        "else { Copy-Item \"$persist_dir\\dottorrent-gui.ini\" \"$dir\" -Force }"
```

- Nagating condition does not make sense when you use both branches.
- Why `Copy-Item` with `-Force` parameter?

```suggestion
    "extract_dir": "dottorrent-gui",
    "pre_install": [
        "ensure \"$persist_dir\" | Out-Null",
        "if (Test-Path \"$persist_dir\\dottorrent-gui.ini\") {",
        "    Copy-Item \"$persist_dir\\dottorrent-gui.ini\" \"$dir\"",
        "} else {",
        "    New-Item \"$dir\\dottorrent-gui.ini\" | Out-Null",
        "}"
```

---

Is there any particular reason to ensure $persist_dir during installation? Doing it during removal/update makes more sense to me since the file is only then copied and thus the folder is actually being used.

---

To be consistent with classic persisting.

Files are directly linked == directory created